src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Commands.xml

```diff
@@ -19,6 +19,18 @@
       <SeeAlso Title="Package source sections in NuGet.config files" Url="/nuget/reference/nuget-config-file#package-source-sections" />
       <SeeAlso Title="sources command (nuget.exe)" Url="/nuget/reference/cli-reference/cli-ref-sources" />
     </Noun>
+    <Noun Name="client-cert">
+      <SingleValueOption Name="package-source" Shortcut="s" Help="Option_PackageSource"/>
+      <SingleValueOption Name="path" Help="Option_Path"/>
+      <SingleValueOption Name="password" Help="Option_Password"/>
+      <SwitchOption Name="store-password-in-clear-text" Help="Option_StorePasswordInClearText"/>
+      <SingleValueOption Name="store-location" Help="Option_StoreLocation"/>
+      <SingleValueOption Name="store-name" Help="Option_StoreName"/>
+      <SingleValueOption Name="find-by" Help="Option_FindBy"/>
+      <SingleValueOption Name="find-value" Help="Option_FindValue"/>
+      <SwitchOption Name="force" Shortcut="f" Help="Option_Force"/>
+      <SingleValueOption Name="configfile" Help="Option_ConfigFile" />
```

Comment:
If appropriate, and it likely is, you should add some examples and seealsos -- to help create doc pages for dotnet sdk doc pages.

Reply:
likely covered by NuGet/Home#9528 - will let @zivkan be the judge/planner on that.

Reply:
@rrelyea  if it's just about docs, let's do it in another PR. I was more concerned about the changes to the `Verbs.tt` file, hoping to get your feedback on that.

Reply:
even if we don't go do the docs now...when we run ttGen...the docs.tt file shouldn't crash.

PS C:\repos\blackgad-nc> .\scripts\utils\ttGen.ps1
Regenerating all generated files from all .tt files.
(requires t4.exe from 'dotnet tool install --global dotnet-t4' or right click .tt files in VS and 'run custom tool')

t4.exe packages\nuget.client.endtoend.testdata\1.0.0\content\Packages\FileTransformWorksOnDependentFile\files\TTFile.1.0.0\content\one.tt
t4.exe src\NuGet.Core\NuGet.CommandLine.XPlat\Commands\CommandParsers.tt
t4.exe src\NuGet.Core\NuGet.CommandLine.XPlat\Commands\Verbs.tt
t4.exe src\NuGet.Core\NuGet.CommandLine.XPlat\external\docs.tt
Processing 'src\NuGet.Core\NuGet.CommandLine.XPlat\external\docs.tt' failed.
ERROR: Error running transform: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ArgumentNullException: Value cannot be null.
Parameter name: objectToConvert
   at Microsoft.VisualStudio.TextTemplating.ToStringHelper.ToStringWithCulture(Object objectToConvert) in /Users/mikayla/code/t4/Mono.TextTemplating/Microsoft.VisualStudio.TextTemplating/ToStringHelper.cs:line 46
   at Microsoft.VisualStudio.TextTemplating.GeneratedTextTransformation2c5c294a.TransformText()
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Mono.TextTemplating.CompiledTemplate.Process() in /Users/mikayla/code/t4/Mono.TextTemplating/Mono.TextTemplating/CompiledTemplate.cs:line 119
t4.exe src\NuGet.Core\NuGet.Commands\CommandArgs\VerbArgs.tt
t4.exe test\EndToEnd\Packages\FileTransformWorksOnDependentFile\files\TTFile.1.0.0\content\one.tt
PS C:\repos\blackgad-nc>

Reply:
done